content/tutorial/tutorial.md

```diff
@@ -1171,7 +1171,7 @@ También reemplazaremos `this.state.history` por `this.state.history.slice(0, th
   }
 ```
 
-Finalmente, modificaremos el método `render` del componente Game de siempre renderizar el último movimiento a renderizar el movimiento seleccionado actualmente de acuerdo a `stepNumber`:
```

Comment:
Hi @Javier2x! Thank you very much for your suggestion. However, I think that with this change we would lose some of the original intent.  

> Finally, we will modify the Game component’s render method from always rendering the last move to rendering the currently selected move according to stepNumber:

The original text chooses to use this repetitive style, which makes sense in the context of a step by step tutorial. Let us know what you think!